pkg/controllers/sync/controller.go

```diff
@@ -596,6 +597,14 @@ func (s *SyncController) syncToClusters(
 			}
 		}
 
+		if clusterObj != nil {
+			// If one of fedObject and clusterObj is a derived service and the other is not,
+			// the obtained clusterObj does not correspond to fedObj and should be set to nil.
+			if util.IsDerivedService(fedResource.Object().GetAnnotations()) != util.IsDerivedService(clusterObj.GetAnnotations()) {
```

Comment:
I'm having a hard time understanding when this would be the case. If we reach this line, the cluster service must have the kubeadmiral managed label and must have been synced by us. Hence they must have the same `derived` status. Could you point out the flaw in my logic?

Reply:
Now there will be two fedObj, their template is the svc with the same name, the first is fedObj generated by sourceObj for foo, the second is fedObj with derived prefix generated by serviceImport for foo, we must make sure to delete the second fedObj (even if it does not take effect due to conflicts), the first type of svc issued will not be cleared. Therefore, it is necessary to provide a mechanism to mark which FedObj issued the svc of the member cluster. Considering that this situation only occurs under mcs, a special label is used.